rust/libnewsboat-ffi/src/utils.rs

```diff
@@ -418,6 +418,20 @@ pub unsafe extern "C" fn rs_strwidth_stfl(input: *const c_char) -> usize {
     })
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn rs_substr_with_width(
+    string: *const c_char,
+    max_width: usize,
+) -> *mut c_char {
+    abort_on_panic(|| {
+        let rs_str = CStr::from_ptr(string);
+        let rs_str = rs_str.to_string_lossy().into_owned();
+        let output = utils::substr_with_width(&rs_str, max_width);
+        let result = CString::new(output).unwrap();
```

Comment:
Please add a comment explaining why this `unwrap` will never panic.

Reply:
Done.